migrations/1618507684019-rename-0-conf.js

```diff
@@ -0,0 +1,24 @@
+const _ = require('lodash/fp')
+const settingsLoader = require('../lib/new-settings-loader')
+const configManager = require('../lib/new-config-manager')
+
+const isNil = val => val == null
+
+exports.up = async function (next) {
+  const settingsPromise = settingsLoader.loadLatest()
+  const [{ config }] = await Promise.all([settingsPromise])
```

Comment:
Isn't the promise redundant? You can just call await with the function which returns the promise, instead of wrapping it around another promise.
Something like this (I'm not sure if the deconstruction remains the same, so test it first):

```suggestion
  const [{ config }] = await settingsLoader.loadLatest()
```